active_shipping.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
 
 
   s.add_dependency("measured", ">= 2.0")
-  s.add_dependency("activesupport", ">= 4.2", "< 5.2.0")
+  s.add_dependency("activesupport", ">= 4.2", "<= 5.2.0.beta2")
```

Comment:
Maybe it is better to remove the upper restriction like we are doing in the projects in the rails organization.

Reply:
BTW, beta2 is already allowed by `< 5.2.0` since for rubygems any pre release is less than the final release.

Reply:
I'm not fond of that policy personally, and yeah I realized this is mostly pointless until 5.2 final is out